ironfish/src/mining/miner.test.ts

```diff
@@ -122,14 +122,14 @@ describe('Miner', () => {
 
 describe('mineHeader', () => {
   beforeEach(() => {
-    mocked(blockHeaderModule.hashBlockHeader).mockReset()
+    mocked(nativeModule.mineHeaderBatch).mockReset()
```

Comment:
Git makes this diff pretty gross, I just re-wrote the block entirely, so might be easier to review without a diff-view